tests/test_config_validation.py

```
@@ -0,0 +1,722 @@
+    code (and validators) need."""
+
+    BASE_FIELDS = [
+        "model_name", "name", "max_width", "hidden_size", "dropout",
+        "fine_tune", "subtoken_pooling", "span_mode", "post_fusion_schema",
+        "num_post_fusion_layers", "vocab_size", "max_neg_type_ratio",
+        "max_types", "max_len", "words_splitter_type", "num_rnn_layers",
+        "fuse_layers", "embed_ent_token", "class_token_index",
+        "encoder_config", "ent_token", "sep_token", "_attn_implementation",
+        "token_loss_coef", "span_loss_coef", "represent_spans",
+        "neg_spans_ratio",
+    ]
```

Ruff (0.15.1)

[warning] Mutable default value for class attribute

(RUF012)